js/interface.js

```diff
@@ -708,27 +708,24 @@ var DynamicLists = (function() {
         type: []
       };
 
-      for ( var type in options ) {
-        if (!options[type]) {
-          continue;
-        }
-
-        var accessRuleIndex = -1;
-
-        defaultRule.type = [type];
-
-        accessRules.forEach(function(item, index) {
-          var typeIndex = item.type.indexOf(type);
+      for (var type in options) {
+        if (options.hasOwnProperty(type)) {
```

Comment:
Why did you change the logic to the other way round?

```suggestion
        if (!options.hasOwnProperty(type)) {
          continue;
        }
```

Reply:
There was:
```suggestion
        if (!options[type]) {
          continue;
        }
```
And it skipped needed iterations when ``options[type] === false`` 
I added `if (options.hasOwnProperty(type)) {` after creating PR because of Codacy error, linter didn't highlight it (version ^1.0.5)

I think there are no difference between `hasOwnProp` and `hasOwnProp + continue`, in both cases iteration will be skipped in a first row after condition evaluation, should I change this?

Reply:
Yes please change it. Fliplet's coding standard references the Airbnb JavaScript style guide, with [this specific rule](https://github.com/airbnb/javascript/blob/master/README.md#blocks--no-else-return). The same logic applies to `continue` as well cc @sofiiakvasnevska The Fliplet coding standard has been updated to specify this, please make sure all developers are aware.